base_location_geonames_import/README.rst

```diff
@@ -26,7 +26,7 @@ system parameter.
 Usage
 =====
 
-Go to *Sales > Configuration > Address book > Localization > Import from Geonames*,
+Go to *Configurations > Technical > Cities/Locations Management > Import from Geonames*,
```

Comment:
You are totally right! I was only correcting the put name without s, but it's not the English name.